addons/nodot/interaction/Teleport3D.gd

```diff
@@ -0,0 +1,101 @@
+##  Teleport3D is an Area3D which, when touched, teleports the collider to another location either instantly or after a defined time
+class_name Teleport3D extends Area3D
+
+@export_category("Nodes")
+##  Target node for copy the coords
```

Comment:
Seems like there's 2 spaces after the hashes. Please refer to the other node code to see how they should be formatted.